src/soilbiogeochem/SoilBiogeochemCarbonFluxType.F90

```diff
@@ -32,6 +35,11 @@ module SoilBiogeochemCarbonFluxType
      real(r8), pointer :: decomp_k_col                              (:,:,:) ! rate constant for decomposition (1./sec)
      real(r8), pointer :: hr_vr_col                                 (:,:)   ! (gC/m3/s) total vertically-resolved het. resp. from decomposing C pools 
      real(r8), pointer :: o_scalar_col                              (:,:)   ! fraction by which decomposition is limited by anoxia
+!KO
```

Comment:
I'm not sure these fluxes are necessary in the code.  Maybe they should remain on a development branch an left out of this pull request?